analytics/github_analyze.py

```diff
@@ -352,12 +356,14 @@ def parse_arguments():
 def main():
     import time
     args = parse_arguments()
-    remotes = get_git_remotes(args.repo_path)
-    # Pick best remote
-    remote = next(iter(remotes.keys()))
-    for key in remotes:
-        if remotes[key].endswith('github.com/pytorch/pytorch'):
-            remote = key
+    remote = args.remote
+    if not remote:
+        remotes = get_git_remotes(args.repo_path)
+        # Pick best remote
+        remote = next(iter(remotes.keys()))
+        for key in remotes:
+            if remotes[key] == 'https://github.com/pytorch/pytorch':
```

Comment:
Please use `endswith` instead of equal (as all my remotes look like `https://malfet:XXXX@github.com/pytorch/pytorch`)
```suggestion
            if remotes[key].endswith('github.com/pytorch/pytorch'):
```